16/09/2021 11:05:38 PM:INFO:Importing Required Dependencies
16/09/2021 11:05:50 PM:INFO:Connecting a Database
16/09/2021 11:05:50 PM:INFO:Reading the Data from MySQL
16/09/2021 11:05:53 PM:INFO:Exploratory Data Analysis
16/09/2021 11:05:54 PM:INFO:Replacing a value with ham as 0 and spam as 1
16/09/2021 11:05:54 PM:INFO:Plotting histogram
16/09/2021 11:05:55 PM:INFO:Independent and Dependent variables
16/09/2021 11:05:55 PM:INFO:Extract Feature With CountVectorizer
16/09/2021 11:05:56 PM:INFO:Dumping Extracted features to pickle file
16/09/2021 11:05:57 PM:INFO:Splitting the data into Train and Test
16/09/2021 11:05:57 PM:INFO:Fitting the Model Using MultinomialNB
16/09/2021 11:05:57 PM:INFO:Fitting the Model Using LogisticRegression
16/09/2021 11:06:00 PM:INFO:Fitting the Model Using AdaBoostClassifier
16/09/2021 11:06:03 PM:INFO:Fitting the Model Using ExtraTreesClassifier
16/09/2021 11:06:12 PM:INFO:Fitting the Model Using GradientBoostingClassifier
16/09/2021 11:06:17 PM:INFO:Fitting the Model Using XGBClassifier
16/09/2021 11:06:18 PM:INFO:Dumping the Model to pickle file
16/09/2021 11:06:18 PM:INFO:Prediction on user data
16/09/2021 11:06:19 PM:INFO:Loading pickle file and Prediction
16/09/2021 11:07:41 PM:INFO:Loading a model and transform file
16/09/2021 11:07:42 PM:INFO:Configure Database
16/09/2021 11:07:42 PM:INFO: * Restarting with windowsapi reloader
16/09/2021 11:07:44 PM:INFO:Loading a model and transform file
16/09/2021 11:07:44 PM:INFO:Configure Database
16/09/2021 11:07:44 PM:WARNING: * Debugger is active!
16/09/2021 11:07:44 PM:INFO: * Debugger PIN: 239-064-683
16/09/2021 11:07:44 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
16/09/2021 11:08:12 PM:INFO:Reading a Message from user
16/09/2021 11:08:12 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:12] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:08:12 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:12] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:08:15 PM:INFO:Rendering a Home page
16/09/2021 11:08:15 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:15] "[37mGET / HTTP/1.1[0m" 200 -
16/09/2021 11:08:16 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:16] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:08:45 PM:INFO:Reading a Message from user
16/09/2021 11:08:45 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:45] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:08:46 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:08:46] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:12:05 PM:INFO:Rendering a Home page
16/09/2021 11:12:05 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:12:05] "[37mGET / HTTP/1.1[0m" 200 -
16/09/2021 11:12:05 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:12:05] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:19:28 PM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\app.py', reloading
16/09/2021 11:19:28 PM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\app.py', reloading
16/09/2021 11:19:29 PM:INFO: * Restarting with windowsapi reloader
16/09/2021 11:19:32 PM:INFO:Loading a model and transform file
16/09/2021 11:19:32 PM:INFO:Configure Database
16/09/2021 11:19:32 PM:WARNING: * Debugger is active!
16/09/2021 11:19:32 PM:INFO: * Debugger PIN: 239-064-683
16/09/2021 11:19:32 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
16/09/2021 11:19:45 PM:INFO:Reading a Message from user
16/09/2021 11:19:45 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:19:45] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:19:45 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:19:45] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:20:44 PM:INFO:Reading a Message from user
16/09/2021 11:20:45 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:20:45] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:20:45 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:20:45] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:23:06 PM:INFO:Reading a Message from user
16/09/2021 11:23:06 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:23:06] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:23:06 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:23:06] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:25:24 PM:INFO:Reading a Message from user
16/09/2021 11:25:24 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:25:24] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:25:24 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:25:24] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:29:35 PM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\app.py', reloading
16/09/2021 11:29:35 PM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\app.py', reloading
16/09/2021 11:29:35 PM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\app.py', reloading
16/09/2021 11:29:36 PM:INFO: * Restarting with windowsapi reloader
16/09/2021 11:29:38 PM:INFO:Loading a model and transform file
16/09/2021 11:29:39 PM:INFO:Configure Database
16/09/2021 11:29:39 PM:WARNING: * Debugger is active!
16/09/2021 11:29:39 PM:INFO: * Debugger PIN: 239-064-683
16/09/2021 11:29:39 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
16/09/2021 11:29:49 PM:INFO:Rendering a Home page
16/09/2021 11:29:49 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:29:49] "[37mGET / HTTP/1.1[0m" 200 -
16/09/2021 11:29:49 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:29:49] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
16/09/2021 11:30:09 PM:INFO:Reading a Message from user
16/09/2021 11:30:09 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:30:09] "[37mPOST /predict HTTP/1.1[0m" 200 -
16/09/2021 11:30:09 PM:INFO:127.0.0.1 - - [16/Sep/2021 23:30:09] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
17/09/2021 12:00:42 AM:INFO:Rendering a Home page
17/09/2021 12:00:42 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:00:42] "[37mGET / HTTP/1.1[0m" 200 -
17/09/2021 12:00:42 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:00:42] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
17/09/2021 12:00:45 AM:INFO:Rendering a Home page
17/09/2021 12:00:45 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:00:45] "[37mGET / HTTP/1.1[0m" 200 -
17/09/2021 12:00:45 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:00:45] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
17/09/2021 12:01:20 AM:INFO:Rendering a Home page
17/09/2021 12:01:20 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:01:20] "[37mGET / HTTP/1.1[0m" 200 -
17/09/2021 12:01:20 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:01:20] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
17/09/2021 12:02:22 AM:INFO:Rendering a Home page
17/09/2021 12:02:22 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:02:22] "[37mGET / HTTP/1.1[0m" 200 -
17/09/2021 12:02:22 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:02:22] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
17/09/2021 12:02:44 AM:INFO:Rendering a Home page
17/09/2021 12:02:44 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:02:44] "[37mGET / HTTP/1.1[0m" 200 -
17/09/2021 12:02:44 AM:INFO:127.0.0.1 - - [17/Sep/2021 00:02:44] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
18/09/2021 06:34:43 PM:INFO:Loading a model and transform file
18/09/2021 06:34:43 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:/Users/prash/Desktop/data/SMS_Spam_Ham_Classification/app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
18/09/2021 06:34:43 PM:INFO:Configure Database
18/09/2021 06:34:43 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:/Users/prash/Desktop/data/SMS_Spam_Ham_Classification/app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
18/09/2021 06:34:44 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:34:45 PM:INFO:Loading a model and transform file
18/09/2021 06:34:45 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
18/09/2021 06:34:45 PM:INFO:Configure Database
18/09/2021 06:34:45 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
18/09/2021 06:34:45 PM:WARNING: * Debugger is active!
18/09/2021 06:34:46 PM:INFO: * Debugger PIN: 430-617-474
18/09/2021 06:34:46 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:34:52 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Scripts\\conda-script.py', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\common\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\common\\__pycache__\\compat.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\cli\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\cli\\__pycache__\\main.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\_vendor\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\conda\\_vendor\\toolz\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\cytoolz\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\itertoolz.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\utils.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\functoolz.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\_signatures.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\dicttoolz.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\recipes.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\curried\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\curried\\__pycache__\\operator.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\curried\\__pycache__\\exceptions.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\sandbox\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\sandbox\\__pycache__\\core.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\sandbox\\__pycache__\\parallel.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\toolz\\__pycache__\\_version.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\cytoolz\\__pycache__\\_signatures.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\cytoolz\\curried\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\cytoolz\\curried\\__pycache__\\operator.cpython-38.pyc', reloading
18/09/2021 06:34:53 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:34:54 PM:INFO:Loading a model and transform file
18/09/2021 06:34:54 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
18/09/2021 06:34:54 PM:INFO:Configure Database
18/09/2021 06:34:54 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
18/09/2021 06:34:54 PM:WARNING: * Debugger is active!
18/09/2021 06:34:54 PM:INFO: * Debugger PIN: 430-617-474
18/09/2021 06:34:54 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:35:03 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\html\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:35:03 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\html\\__pycache__\\entities.cpython-38.pyc', reloading
18/09/2021 06:35:03 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\__pycache__\\socketserver.cpython-38.pyc', reloading
18/09/2021 06:35:04 PM:INFO:Loading a model and transform file
18/09/2021 06:35:05 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:35:06 PM:INFO:Loading a model and transform file
18/09/2021 06:35:06 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
18/09/2021 06:35:06 PM:INFO:Configure Database
18/09/2021 06:35:06 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
18/09/2021 06:35:06 PM:WARNING: * Debugger is active!
18/09/2021 06:35:06 PM:INFO: * Debugger PIN: 430-617-474
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\__pycache__\\stringprep.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\_lib\\__pycache__\\_testutils.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\_lib\\__pycache__\\deprecation.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\__pycache__\\__config__.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\__pycache__\\version.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\__pycache__\\_distributor_init.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\_lib\\__pycache__\\_pep440.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\_lib\\__pycache__\\_ccallback.cpython-38.pyc', reloading
18/09/2021 06:35:06 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\fft\\__pycache__\\__init__.cpython-38.pyc', reloading
18/09/2021 06:35:07 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\fft\\__pycache__\\_basic.cpython-38.pyc', reloading
18/09/2021 06:35:07 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\scipy\\_lib\\__pycache__\\uarray.cpython-38.pyc', reloading
18/09/2021 06:35:08 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:35:10 PM:INFO:Loading a model and transform file
18/09/2021 06:35:10 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
18/09/2021 06:35:10 PM:INFO:Configure Database
18/09/2021 06:35:10 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
18/09/2021 06:35:10 PM:WARNING: * Debugger is active!
18/09/2021 06:35:10 PM:INFO: * Debugger PIN: 430-617-474
18/09/2021 06:35:10 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\hashing.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\func_inspect.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\logger.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\disk.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\multiprocessing\\__pycache__\\util.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\_store_backends.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\backports.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\numpy_pickle.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\compressor.cpython-38.pyc', reloading
18/09/2021 06:35:10 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\numpy_pickle_utils.cpython-38.pyc', reloading
18/09/2021 06:35:11 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\numpy_pickle_compat.cpython-38.pyc', reloading
18/09/2021 06:35:11 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\parallel.cpython-38.pyc', reloading
18/09/2021 06:35:11 PM:INFO: * Detected change in 'C:\\Users\\prash\\Anaconda3\\Lib\\site-packages\\joblib\\__pycache__\\_multiprocessing_helpers.cpython-38.pyc', reloading
18/09/2021 06:35:12 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:35:50 PM:INFO:Configure Database
18/09/2021 06:35:50 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:35:51 PM:INFO:Loading a model and transform file
18/09/2021 06:35:52 PM:INFO:Configure Database
18/09/2021 06:35:52 PM:WARNING: * Debugger is active!
18/09/2021 06:35:52 PM:INFO: * Debugger PIN: 239-064-683
18/09/2021 06:35:52 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:36:13 PM:INFO:Rendering a Home page
18/09/2021 06:36:13 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:36:13] "[37mGET / HTTP/1.1[0m" 200 -
18/09/2021 06:36:14 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:36:14] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
18/09/2021 06:36:15 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:36:15] "[36mGET /static/spam-filter.jpg HTTP/1.1[0m" 304 -
18/09/2021 06:36:16 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:36:16] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
18/09/2021 06:36:56 PM:INFO:Reading a Message from user
18/09/2021 06:36:57 PM:INFO: * Detected change in 'C:\\Users\\prash\\anaconda3\\Lib\\site-packages\\flask\\__pycache__\\debughelpers.cpython-38.pyc', reloading
18/09/2021 06:36:57 PM:INFO: * Detected change in 'C:\\Users\\prash\\anaconda3\\Lib\\site-packages\\MySQLdb\\__pycache__\\connections.cpython-38.pyc', reloading
18/09/2021 06:36:58 PM:INFO: * Restarting with windowsapi reloader
18/09/2021 06:36:59 PM:INFO:Loading a model and transform file
18/09/2021 06:37:01 PM:INFO:Configure Database
18/09/2021 06:37:01 PM:WARNING: * Debugger is active!
18/09/2021 06:37:01 PM:INFO: * Debugger PIN: 239-064-683
18/09/2021 06:37:01 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
18/09/2021 06:37:11 PM:INFO:Reading a Message from user
18/09/2021 06:37:11 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:37:11] "[37mPOST /predict HTTP/1.1[0m" 200 -
18/09/2021 06:37:11 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:37:11] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
18/09/2021 06:37:58 PM:INFO:Reading a Message from user
18/09/2021 06:37:58 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:37:58] "[37mPOST /predict HTTP/1.1[0m" 200 -
18/09/2021 06:37:58 PM:INFO:127.0.0.1 - - [18/Sep/2021 18:37:58] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
18/09/2021 07:08:20 PM:INFO:127.0.0.1 - - [18/Sep/2021 19:08:20] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
18/09/2021 07:13:16 PM:INFO:Reading a Message from user
18/09/2021 07:13:16 PM:INFO:127.0.0.1 - - [18/Sep/2021 19:13:16] "[37mPOST /predict HTTP/1.1[0m" 200 -
18/09/2021 07:13:16 PM:INFO:127.0.0.1 - - [18/Sep/2021 19:13:16] "[33mGET /static/css/styles.css HTTP/1.1[0m" 404 -
19/09/2021 10:40:48 AM:INFO: * Detected change in 'C:\\Users\\prash\\Desktop\\data\\SMS_Spam_Ham_Classification\\SMS_Ham_Spam_Classification.py', reloading
19/09/2021 10:40:54 AM:INFO: * Restarting with windowsapi reloader
19/09/2021 10:41:21 AM:INFO:Loading a model and transform file
19/09/2021 10:41:54 AM:INFO:Configure Database
19/09/2021 10:41:54 AM:WARNING: * Debugger is active!
19/09/2021 10:41:55 AM:INFO: * Debugger PIN: 239-064-683
19/09/2021 10:41:55 AM:INFO: * Detected change in 'C:\\Users\\prash\\anaconda3\\Lib\\encodings\\__pycache__\\idna.cpython-38.pyc', reloading
19/09/2021 10:41:55 AM:INFO: * Detected change in 'C:\\Users\\prash\\anaconda3\\Lib\\__pycache__\\stringprep.cpython-38.pyc', reloading
19/09/2021 10:41:55 AM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
19/09/2021 10:41:57 AM:INFO: * Restarting with windowsapi reloader
19/09/2021 10:41:59 AM:INFO:Loading a model and transform file
19/09/2021 10:42:01 AM:INFO:Configure Database
19/09/2021 10:42:01 AM:WARNING: * Debugger is active!
19/09/2021 10:42:01 AM:INFO: * Debugger PIN: 239-064-683
19/09/2021 10:42:02 AM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
21/09/2021 01:33:40 PM:INFO:Importing Required Dependencies
21/09/2021 01:34:45 PM:INFO:Connecting a Database
21/09/2021 01:34:47 PM:INFO:Reading the Data from MySQL
21/09/2021 01:34:51 PM:INFO:Exploratory Data Analysis
21/09/2021 01:34:52 PM:INFO:Replacing a value with ham as 0 and spam as 1
21/09/2021 01:34:54 PM:INFO:Plotting histogram
21/09/2021 01:34:55 PM:INFO:Independent and Dependent variables
21/09/2021 01:34:57 PM:INFO:Extract Feature With CountVectorizer
21/09/2021 01:34:58 PM:INFO:Dumping Extracted features to pickle file
21/09/2021 01:34:59 PM:INFO:Splitting the data into Train and Test
21/09/2021 01:35:01 PM:INFO:Fitting the Model Using MultinomialNB
21/09/2021 01:35:02 PM:INFO:Fitting the Model Using LogisticRegression
21/09/2021 01:35:03 PM:INFO:Fitting the Model Using AdaBoostClassifier
21/09/2021 01:35:05 PM:INFO:Fitting the Model Using ExtraTreesClassifier
21/09/2021 01:35:15 PM:INFO:Fitting the Model Using GradientBoostingClassifier
21/09/2021 01:35:19 PM:INFO:Fitting the Model Using XGBClassifier
21/09/2021 01:51:38 PM:INFO:Fitting the Model Using ExtraTreesClassifier
21/09/2021 01:52:08 PM:INFO:Fitting the Model Using ExtraTreesClassifier
21/09/2021 01:54:27 PM:INFO:Fitting the Model Using ExtraTreesClassifier
21/09/2021 01:55:23 PM:INFO:Fitting the Model Using ExtraTreesClassifier
21/09/2021 01:55:59 PM:INFO:Fitting the Model Using ExtraTreesClassifier
22/09/2021 08:16:00 PM:INFO:Importing Required Dependencies
22/09/2021 08:17:16 PM:INFO:Connecting a Database
22/09/2021 08:17:19 PM:INFO:Reading the Data from MySQL
22/09/2021 08:17:22 PM:INFO:Exploratory Data Analysis
22/09/2021 08:17:22 PM:INFO:Replacing a value with ham as 0 and spam as 1
22/09/2021 08:17:23 PM:INFO:Plotting histogram
22/09/2021 08:17:26 PM:INFO:Independent and Dependent variables
22/09/2021 08:17:27 PM:INFO:Extract Feature With CountVectorizer
22/09/2021 08:17:29 PM:INFO:Dumping Extracted features to pickle file
22/09/2021 08:17:30 PM:INFO:Splitting the data into Train and Test
22/09/2021 08:17:40 PM:INFO:Fitting the Model Using MultinomialNB
22/09/2021 08:17:40 PM:INFO:Fitting the Model Using LogisticRegression
22/09/2021 08:17:42 PM:INFO:Fitting the Model Using AdaBoostClassifier
22/09/2021 08:17:44 PM:INFO:Fitting the Model Using ExtraTreesClassifier
22/09/2021 08:17:59 PM:INFO:Fitting the Model Using GradientBoostingClassifier
22/09/2021 08:18:05 PM:INFO:Fitting the Model Using XGBClassifier
22/09/2021 08:18:54 PM:INFO:Getting best Parameters
22/09/2021 08:20:16 PM:INFO:Fitting the model using Hyper parameters 
22/09/2021 08:20:26 PM:INFO:Dumping the Model to pickle file
22/09/2021 08:20:28 PM:INFO:Prediction on user data
22/09/2021 08:20:30 PM:INFO:Loading pickle file and Prediction
22/09/2021 08:24:28 PM:INFO:Loading a model and transform file
22/09/2021 08:24:28 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:/Users/prash/Desktop/data/SMS_Spam_Ham_Classification/app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
22/09/2021 08:24:28 PM:INFO:Configure Database
22/09/2021 08:24:28 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:/Users/prash/Desktop/data/SMS_Spam_Ham_Classification/app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
22/09/2021 08:24:30 PM:INFO: * Restarting with windowsapi reloader
22/09/2021 08:24:31 PM:INFO:Loading a model and transform file
22/09/2021 08:24:31 PM:ERROR:[Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 14, in <module>
    model = pickle.load(open(filename, 'rb'))
FileNotFoundError: [Errno 2] No such file or directory: 'Extratrees_classifier.pkl'
22/09/2021 08:24:31 PM:INFO:Configure Database
22/09/2021 08:24:31 PM:ERROR:[Errno 2] No such file or directory: 'db.yaml'
Traceback (most recent call last):
  File "c:\Users\prash\Desktop\data\SMS_Spam_Ham_Classification\app.py", line 25, in <module>
    db = yaml.load(open('db.yaml'))
FileNotFoundError: [Errno 2] No such file or directory: 'db.yaml'
22/09/2021 08:24:31 PM:WARNING: * Debugger is active!
22/09/2021 08:24:31 PM:INFO: * Debugger PIN: 430-617-474
22/09/2021 08:24:31 PM:INFO: * Running on http://127.0.0.1:5000/ (Press CTRL+C to quit)
22/09/2021 08:25:25 PM:INFO:Importing Required Dependencies
22/09/2021 08:26:05 PM:INFO:Connecting a Database
22/09/2021 08:26:05 PM:INFO:Reading the Data from MySQL
22/09/2021 08:26:08 PM:INFO:Exploratory Data Analysis
22/09/2021 08:26:08 PM:INFO:Replacing a value with ham as 0 and spam as 1
22/09/2021 08:26:08 PM:INFO:Plotting histogram
22/09/2021 08:26:47 PM:INFO:Independent and Dependent variables
22/09/2021 08:26:47 PM:INFO:Extract Feature With CountVectorizer
22/09/2021 08:26:48 PM:INFO:Dumping Extracted features to pickle file
22/09/2021 08:26:48 PM:INFO:Splitting the data into Train and Test
22/09/2021 08:26:48 PM:INFO:Fitting the Model Using MultinomialNB
22/09/2021 08:26:48 PM:INFO:Fitting the Model Using LogisticRegression
22/09/2021 08:26:48 PM:INFO:Fitting the Model Using AdaBoostClassifier
22/09/2021 08:26:52 PM:INFO:Fitting the Model Using ExtraTreesClassifier
22/09/2021 08:27:09 PM:INFO:Fitting the Model Using GradientBoostingClassifier
22/09/2021 08:27:16 PM:INFO:Fitting the Model Using XGBClassifier
22/09/2021 08:27:19 PM:INFO:Getting best Parameters
22/09/2021 08:27:33 PM:INFO:Fitting the model using Hyper parameters 
22/09/2021 08:27:38 PM:INFO:Dumping the Model to pickle file
22/09/2021 08:27:38 PM:INFO:Prediction on user data
22/09/2021 08:27:38 PM:INFO:Loading pickle file and Prediction
22/09/2021 08:30:23 PM:INFO:Importing Required Dependencies
22/09/2021 08:30:28 PM:INFO:Connecting a Database
22/09/2021 08:30:29 PM:INFO:Reading the Data from MySQL
22/09/2021 08:30:29 PM:INFO:Exploratory Data Analysis
22/09/2021 08:30:29 PM:INFO:Replacing a value with ham as 0 and spam as 1
22/09/2021 08:30:29 PM:INFO:Plotting histogram
22/09/2021 08:30:37 PM:INFO:Independent and Dependent variables
22/09/2021 08:30:37 PM:INFO:Extract Feature With CountVectorizer
22/09/2021 08:30:38 PM:INFO:Dumping Extracted features to pickle file
22/09/2021 08:30:38 PM:INFO:Splitting the data into Train and Test
22/09/2021 08:30:38 PM:INFO:Fitting the Model Using MultinomialNB
22/09/2021 08:30:38 PM:INFO:Fitting the Model Using LogisticRegression
22/09/2021 08:30:38 PM:INFO:Fitting the Model Using AdaBoostClassifier
22/09/2021 08:30:42 PM:INFO:Fitting the Model Using ExtraTreesClassifier
22/09/2021 08:31:02 PM:INFO:Fitting the Model Using GradientBoostingClassifier
22/09/2021 08:31:11 PM:INFO:Fitting the Model Using XGBClassifier
22/09/2021 08:31:14 PM:INFO:Getting best Parameters
22/09/2021 08:31:33 PM:INFO:Fitting the model using Hyper parameters 
22/09/2021 08:31:38 PM:INFO:Dumping the Model to pickle file
22/09/2021 08:31:38 PM:INFO:Prediction on user data
22/09/2021 08:31:38 PM:INFO:Loading pickle file and Prediction
